Add documentation (#49)
genomic-medicine-sweden/tomte: docs/output.md

</details>

### Transcript quantification

#### Salmon

[`Salmon`](https://salmon.readthedocs.io/en/latest/) quantifies reads.

<details markdown="1">
<summary>Output files</summary>

- `alignment/sample`
  - `quant.sf`: quantification file.
  - `quant.genes.sf`: quantification file per gene.
  - `logs/salmon_quant.log`: log file.
  - `cmd_info.json`: main command line parameters with which Salmon was run.

</details>

### Allignment

#### STAR

[`STAR`](https://github.com/alexdobin/STAR) aligns reads to the genome reference. For further reading and documentation see the [STAR manual](https://physiology.med.cornell.edu/faculty/skrabanek/lab/angsd/lecture_notes/STARmanual.pdf).

<details markdown="1">
<summary>Output files</summary>

- `alignment/`
  - `*.SJ.out.tab`: the high confidence collapssed junctions.
  - `*.ReadsPerGene.out.tab`: read count per gene.
  - `*.Log.progress.out`: run progress statistics report updated every minute.
  - `*.Log.out`: log file containing run details.
  - `*.Log.final.out`: a summary of the mapping statistics. It is calculated indivisually per read and then averaged.
  - `*.Aligned.out.bam`: Aligned reads.

</details>

### Tracks

#### Tracks

> Outputs boht junction tracks and bigwig files. For wigToBigWig [`UCSC wigToBigWig`](https://genome.ucsc.edu/goldenPath/help/bigWig.html) is used.

Reviewer: ```suggestion
Outputs both junction tracks and bigwig files. For wigToBigWig [`UCSC wigToBigWig`](https://genome.ucsc.edu/goldenPath/help/bigWig.html) is used.
```

<details markdown="1">
<summary>Output files</summary>

- `analyse_transcripts/drop`
  - `OUTRIDER_provided_samples_top_hits.tsv`: provides at least the top 20 most significant events reported by OUTRIDER in each sample.
  - `OUTRIDER_provided_samples_top_hits_filtered.tsv`: filters OUTRIDER_provided_samples_top_hits according to genes provided by gene_panel_clinical_filter.
  - `FRASER_provided_samples_top_hits.tsv`: provides the aberrant spliced events reported by FRASER.
  - `FRASER_provided_samples_top_hits_filtered.tsv`: filters FRASER_provided_samples_top_hits according to genes provided by gene_panel_clinical_filter.

</details>

#### StringTie

[`StringTie`](https://ccb.jhu.edu/software/stringtie/) will perform guided transcript assembly.

<details markdown="1">
<summary>Output files</summary>

- `analyse_transcripts`
  - `*.coverage.gtf`: coverage on the sample.
  - `*.gene.abundance.txt`: gene abundance on the sample.
  - `*.transcripts.gtf`: transcripts assembled on the sample

</details>

#### GffCompare

> [`GffCompare`](https://github.com/gpertea/gffcompare) annotates stringtie results with the reference marking them each transcript as normal or aberrant.

Reviewer: ```suggestion
[`GffCompare`](https://github.com/gpertea/gffcompare) annotates stringtie results with the reference transcripts, marking each assembled transcript as either normal or aberrant.
```

<details markdown="1">
<summary>Output files</summary>

- `analyse_transcripts`
  - `*.stats`: data summary and accuracy estimation.
  - `*.annotated.gtf`: annotated gtf file.
  - `*.tracking`: transcripts assembled on the sample
  - `*.transcripts.gtf.refmap`: list for each reference transcript what query transcript partially or fully matches it.

>   - `*.transcripts.gtf.tmap`: list the most similar reference trnascript to each query transcript.

Reviewer: ```suggestion
  - `*.transcripts.gtf.tmap`: list the most similar reference transcript to each query transcript.
```

</details>

#### GATK best practices SNV Calling

[`GATK best practices SNV Calling`](https://gatk.broadinstitute.org/hc/en-us/articles/360035531192-RNAseq-short-variant-discovery-SNPs-Indels-) SNV calling will only be activated by setting parameter variant_caller
to "gatk". Involves several steps: [`SplitN Cigar Reads`](https://gatk.broadinstitute.org/hc/en-us/articles/360036858811-SplitNCigarReads), [`Haplotype Caller`](https://gatk.broadinstitute.org/hc/en-us/articles/360037225632-HaplotypeCaller), [`Variant Filtration`](https://gatk.broadinstitute.org/hc/en-us/articles/360037434691-VariantFiltration) and [`BCFtools stats`](https://samtools.github.io/bcftools/bcftools.html#stats).

<details markdown="1">
<summary>Output files</summary>

- `call variants`
  - `*.vcf.gz`: file in vcf format containing variants found in the patient.
  - `*.vcf.gz.tbi`: index for .vcf.gz file.
  - `*.bcftools_stats.txt`: stats on non-reference allele frequency, depth distribution, stats by quality and per-sample counts, singleton stats, etc.

</details>

> ### Allele specific variant Calling

Reviewer: ```suggestion
### Allele specific variant calling
```
